README.md

```diff
@@ -1 +1,84 @@
-alexa-skills-lambda
+Alexa Guardian Skill
+==================== 
+
+Alexa is
+--------
+
+The Alexa Guardian Skill is an Amazon Echo application which allows users to query a part of the Guardian content through Echo. 	
+
+Architecture
+------------
+
+The Skill is implemented as a node application (written in pure JavaScript) running inside a AWS lambda.
+
+
+Prerequisites
+---------------
+
+To run Alexa locally, make sure that you have the following dependencies installed: 
+
+- Node.js version 4.3. To get this version, make sure that you have [nvm](https://github.com/creationix/nvm) installed and run `nvm install 4.3`.
+
+Run `npm install` to install the dependencies. 
+
+Copy the file **config-sample.json** into **tmp/config.json** (You will probably have to create the folder **tmp**.). Replace "YOUR-CAPI-KEY" with your CAPI key. Note that the account type associated to this key should be "External". If you do not have a CAPI key or if your key is not "External" ask the CAPI team to make a key for you.
+
+Running the tests
+-----------------
+
+You can now run `node test` to run the test suite. 
+
+
+Deploying
+---------
+
+Deployment is done with Riff-Raff. 
+
+- project: `content-api::alexa-skills-lambda`
+- stage: `CODE` or `PROD`
+
+Amazon Developer Account
+------------------------
+
+In order to access the online configuration panel for Alexa, you need to create a AWS account (with your Guardian email account). Then ask somebody in the team to add you. 
+
+
+Amazon Certification Process
+----------------------------
+
+The Amazon Certification Process is the mechanism by which newer versions of the Skill configuration (aka "Interaction Model") are sent for approval. Submitting a new version of the configuration is done through your Amazon Developer Account.
+
+
+Current Limitations
+-------------------
+
+1. Update the interaction model can only be done through the developer account. 
+
+2. The actual audio testing needs to be done against the Echo device.
+
+
```

Comment:
Another limitation at current is our ability to accurately/richly search for tags by different aliases meaning we currently have tags and their aliases hard coded in the application. @oilnam  should be able to discuss this with you and why it's a problem.